src/bindings/javascript/analysermodel.cpp

```diff
@@ -7,6 +7,8 @@ using namespace emscripten;
 
 EMSCRIPTEN_BINDINGS(libcellml_analysermodel)
 {
+    register_vector<libcellml::AnalyserEquationPtr>("VectorAnalyserEquation");
+
     enum_<libcellml::AnalyserModel::Type>("TypeM")
```

Comment:
Why `TypeM` and not `AnalyserModel.Type` to mimic what you have done in `src/bindings/javascript/analyserequation.cpp`? (Same for `src/bindings/javascript/analyserequationast.cpp ` and `src/bindings/javascript/analyservariable.cpp`.)

Reply:
Because this is dealing with the analyser equation tests and not analyser model.  That gets sorted in the appropriate PR.

Reply:
Ok